test/pirls.jl

```diff
@@ -7,21 +7,21 @@ using MixedModels: dataset
 
 const gfms = Dict(
     :cbpp => [@formula((incid/hsz) ~ 1 + period + (1|herd))],
-    :contra => [@formula(use ~ 1+age+abs2(age)+urban+livch+(1|urbdist))],
+    :contra => [@formula(use ~ 1+age+abs2(age)+urban+livch+(1|urban&dist))],
     :grouseticks => [@formula(ticks ~ 1+year+ch+ (1|index) + (1|brood) + (1|location))],
     :verbagg => [@formula(r2 ~ 1+anger+gender+btype+situ+(1|subj)+(1|item))],
 )
 
-@testset "contra" begin
+#@testset "contra" begin
```

Comment:
any reason you removed the testset grouping here?

Reply:
If I am running the tests by hand then I comment out the `@testset` and the `end` so that I can step through the test with `<META>-<ENTER>` in VS Code and I don't need to juggle two windows.  So that is vestigial and I will remove it.